### PR TITLE
:sparkles: Add Create New Files and Merge flags

### PR DESCRIPTION
**Pull request by limaneto**
Thos are the ones I've beeing missing.

### CODE REVIEW

**Comment by slashsbin** [comment]
📰  is now for "new files", do you like it too?
I agree with 🤝  for merges.

**Comment by limaneto** [comment]
Yeah that's okay.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Suggested Emojis
 | :pencil: | `:pencil:` | when **performing minor changes/fixing** the code or language |
 | :racehorse: | `:racehorse:` | when improving **performance** |
 | :books: | `:books:` | when writing **docs** |
+| :page_facing_up: | `:page_facing_up:` | when **add new file** to the project |
```
Comment (slashsbin) [inline README.md]: 📰  is for new files now

**Review by slashsbin — CHANGES_REQUESTED** [review]
so please remove 📄  to merge the pull request.

**Pull request merged**